pkg/tui/lists.go

```diff
@@ -18,50 +19,53 @@ var (
 )
 
 const (
-	focusOnList1 = 0
-	focusOnList2 = 1
-	focusOnList3 = 2
+	IssuesList   = 0
+	ProjectsList = 1
+	EpicsList    = 2
 )
 
-func (m *model) createAssignedIssuesList(client *jira.Client, defaults list.DefaultDelegate) *model {
-	issues, _ := getAssignedIssues(client)
+func (m *model) createAssignedIssuesList(defaults list.DefaultDelegate, wg *sync.WaitGroup) *model {
+	defer wg.Done()
+	issues, _ := getAssignedIssues(m.client)
```

Comment:
Not going to handle the potential error returned here?

Reply:
@theoboldalex https://youtube.com/shorts/p-hOKFr9qGY?si=T_7s78wDhPHxsyfy

Reply:
in all seriousness probably needs a refactor, it never actually throws an error only logs it, will refactor those select functions

Reply:
@theoboldalex updated